src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -286,7 +294,7 @@ function getLogoSrcs(changes: TransactionChanges): string[] {
     changes.TokenTransfer.forEach((tokenChange) => logoSet.add(tokenChange.asset.project?.logo?.url))
     changes.TokenApproval.forEach((tokenChange) => logoSet.add(tokenChange.asset.project?.logo?.url))
   }
-  return Array.from(logoSet).filter(Boolean) as string[]
+  return Array.from(logoSet)
```

Comment:
we actually want to include undefined entries in the array of logo srcs, so that PortfolioLogo can determine when to render a missing token logo

|    Before    |   After    |
| ------------ | ------------ |
| <img width="310" alt="image" src="https://github.com/Uniswap/interface/assets/39385577/a33603e2-380d-4bc1-bed8-fba9e0f6493e"> | <img width="318" alt="image" src="https://github.com/Uniswap/interface/assets/39385577/ec732d1b-03f1-4977-ba93-f7abf153a9de"> |